stories/Alert.js

```diff
@@ -15,9 +15,9 @@ storiesOf('Alert', module)
                     hasAction
                         ? {
                               label: 'Dismiss',
-                              onClick: () => {
-                                  // eslint-disable-next-line no-alert
-                                  window.alert('Alright');
+                              onClick: e => {
+                                  // eslint-disable-next-line no-console
+                                  console.log('Clicked alert action', e);
```

Comment:
Seems like using `console.log`in ./stories directory is a valid thing, so we can just set a specific rules for this folder by creating the following file:

**./stories/.eslintrc.json**
```json
{
    "overrides": [{
        "files": [ "*.js" ],
        "rules": {
            "no-console": 0
        }
    }]
}
```

Reply:
Yes that makes more sense, will do